JPS_Ontology/ontology/ontolab/OntoLab.csv

```diff
@@ -3,7 +3,7 @@ OntoLab,TBox,https://www.theworldavatar.com/kg/ontolab/,https://www.w3.org/2007/
 OntoLab,TBox,1.9,http://www.w3.org/2002/07/owl#versionInfo,,,,,,
```

Comment:
Could you please bump up the TBox version number and regenerate the OWL file? Everything else looks fine.

Reply:
Hi Jiaru, I have bump up the version number and regenerated the OWL file. Please let me know if anything else needs to be changed.